packages/@roots/bud-build/src/Build/config.ts

```diff
@@ -52,18 +52,11 @@ export function config(this: Framework): void {
 
   this.hooks
     .on('build/bail', true)
-    .hooks.on('build/cache/name', () => undefined)
-    .hooks.on('build/cache/version', () => undefined)
-    .hooks.on('build/cache/type', () => 'memory')
-    .hooks.on('build/cache/cacheDirectory', () => undefined)
-    .hooks.on('build/cache/cacheLocation', () => undefined)
-    .hooks.on('build/cache/buildDependencies', undefined)
-    .hooks.on('build/cache/managedPaths', () => undefined)
     .hooks.on('build/experiments', () => ({
       lazyCompilation: false,
     }))
     .hooks.on('build/infrastructureLogging', () => ({
-      level: 'none',
```

Comment:
I wasn't certain about whether or not `infrastructureLogging` being explicitly disabled might be implicated by this issue. If it works with `level: none` I would still prefer it for speed improvements. But I don't know enough about this api to feel comfortable.

To prevent `infrastructureLogging` from clogging the console I pass it the bud logging instance. Haven't confirmed if this actually works but it does prevent console output and that's all I really care about right now.